plugins/linux/ifplugin/linuxcalls/admin_linuxcalls.go

```diff
@@ -33,36 +33,29 @@ package linuxcalls
 import (
 	"time"
 
-	"github.com/ligato/cn-infra/logging/measure"
 	"github.com/vishvananda/netlink"
 )
 
 // InterfaceAdminDown calls Netlink API LinkSetDown.
-func InterfaceAdminDown(ifName string, timeLog measure.StopWatchEntry) error {
-	start := time.Now()
-	defer func() {
-		if timeLog != nil {
-			timeLog.LogTimeEntry(time.Since(start))
-		}
-	}()
+func (handler *netLinkHandler) InterfaceAdminDown(ifName string) error {
```

Comment:
Why do these up/down calls contain the **Admin** part? There is no mention of any "admin" in netlink API or anywhere else. Seems like it was inherited from VPP API. We should get rid of it and just keep `InterfaceSetDown` and `InterfaceSetUp` to closer match netlink API.